index.html

```diff
@@ -0,0 +1,1342 @@
+                        <i class="fa fa-chevron-right" aria-hidden="true"></i></button>
+                    <i class="menu fa fa-bars fa-lg d-lg-none" aria-hidden="true"></i>
+                </div>
+            </div>
+
+            <div class="d-none d-lg-block news-category  position-relative">
+                <ul class="list-unstyled d-flex flex-wrap">
+                    <li class="news-cat-li"><a href="#">U.S</a></li>
+                    <li class="news-cat-li"><a href="#">World</a></li>
+                    <li class="news-cat-li"><a href="#">Business</a></li>
+                    <li class="news-cat-li"><a href="#">Tech & Science</a></li>
+                    <li class="news-cat-li"><a href="#">Culture</a></li>
+                    <li class="news-cat-li"><a href="#">Newsgeek</a></li>
+                    <li class="news-cat-li"><a href="#">Sports</a></li>
+                    <li class="news-cat-li"><a href="#">Health</a></li>
+                    <li class="news-cat-li"><a href="#">The Debate</a></li>
+                    <li class="news-cat-li"><a href="#">Experts</a></li>
+                    <li class="news-cat-li"><a href="#">Vantage</a></li>
+                    <li class="search-li"><a href="#">Search <i class="fa fa-search" aria-hidden="true"></i></a>
+                    </li>
+
+                </ul>
+            </div>
+            <div class="d-none d-lg-block trending-news">
+                <ul class="list-unstyled d-flex flex-wrap">
+                    <li><a href="#" class="trending-news-li">TRENDING</a></li>
+                    <li><a href="#" class="trending-news-li">JOE BIDEN</a></li>
+                    <li><a href="#" class="trending-news-li">DONALD TRUMP</a></li>
+                    <li><a href="#" class="trending-news-li">INUGURATIONS</a></li>
+                    <li><a href="#" class="trending-news-li">CORONAVIRUS</a></li>
+                    <li><a href="#" class="trending-news-li">STIMULUS PACKAGE</a></li>
+                    <li><a href="#" class="trending-news-li">VACINE</a></li>
+                    <li><a href="#" class="trending-news-li">CAPITOL RIOTS</a></li>
+                </ul>
+            </div>
+        </nav>
+         
+    </header>
+
+    <div class="main row mx-1">
+        <section class="feature-stories col-lg-3 col-md-5 ">
+            <div class='my-story d-md-none'>
+                <h1 class="text-secondary my-1">TOP STORY</h1>
+                <div class="card border-top-0 border-end-0 border-start-0">
+                    <div class="position-relative">
+                        <img class="card-img-top " src="./assets/images/joe-biden-kamala-harris-their-spouses.jpg"
+                            alt="">
+                        <h5 class="card-title cat-header position-absolute bottom-0 start-0 py-1 px-2 bg-light m-0 text-danger fw-bolder">
+                            Politics</h5>
+                    </div>
```

Comment:
- [ ] [OPTIONAL] If you want to be even more semantic, use the `article` tag for each article inside your sections. Read more on the [article tag here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article).